### DISCUSSION TITLE
JS Docs for Helpers for Serverless Node.js functions

### DISCUSSION THREAD

**Discussion by bigansh** [vercel/vercel#6675] [Help]
Hey, so I created a few serverless functions & now I am facing issues with the available options with the helpers. Is there a way to use JS Docs with these functional params for better IntelliSense & autocomplete? 

Here is what I am talking about! Something like this will be helpful.
```javascript
const vercelHelpers = require('node/vercel')

/**
 * An index function.
 * 
 * @param {VercelHelpers.Request} req 
 * @param {VercelHelpers.Response} res 
 */

module.exports = (req, res) => {
    res.send('index file in serverless functions!')
}
```

**Comment by leerob** [comment]
Are you not interested in using TypeScript for this?

**Reply by bigansh** [reply]
I can use TypeScript, but I thought the ability to do it with JS is also intriguing.
